tests/unit/services/test_rbac_service.py

```
@@ -0,0 +1,109 @@
+    perms_stale = await rbac.get_user_permissions(user_id)
+    assert "users:list" not in perms_stale
+
+    rbac.invalidate(user_id)
+
+    perms_after = await rbac.get_user_permissions(user_id)
+    assert "users:list" in perms_after
+
+
+@pytest.mark.asyncio
+async def test_role_override_bypasses_cache(setup):
+    SessionLocal, admin_id, user_id = setup
```

Ruff (0.15.12)

[warning] Unpacked variable `user_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)